config-model/src/main/java/com/yahoo/vespa/model/container/xml/ContainerModelBuilder.java

```diff
@@ -565,6 +566,14 @@ private void addModelEvaluation(Element spec, ApplicationContainerCluster cluste
         cluster.setModelEvaluation(new ContainerModelEvaluation(cluster, profiles));
     }
 
+    protected void addModelEvaluationBundles(ApplicationContainerCluster cluster) {
+        /* These bundles are added to all application container clusters, even if they haven't
+         * declared 'model-evaluation' in services.xml, because there are many public API packages
+         * in the model-evaluation bundle that could be used by customer code. */
+        cluster.addPlatformBundle(ContainerModelEvaluation.MODEL_EVALUATION_BUNDLE_FILE);
+        cluster.addPlatformBundle(ContainerModelEvaluation.MODEL_INTEGRATION_BUNDLE_FILE);
```

Comment:
I think we should only include model-evaluation by default?

Reply:
Discussed this with @lesters. `model-evaluation` has many publicApi packages, and the code requires model-integration to run. So it seems unsafe to only install the smaller bundle.

Reply:
BTW, there is a plan to remove tensorflow from model-integration, which will reduce its size considerably.